### PR TITLE
fix: add callback for when recording fails

### PR DESCRIPTION
**Pull request by jenniferarnesen**
The callback does not trigger in cases of 40x from the backend.

Update:
image

https://user-images.githubusercontent.com/6113918/132322509-51ff5e06-16b1-4d3a-855a-c09c9dbab62e.mov

### CODE REVIEW

**Review by edoardo — APPROVED** [review]

File: src/pages/view/TitleBar/ActionsBar.js [hunk 1]
```diff
@@ -70,29 +70,35 @@ const ViewActions = ({
         return <Redirect to={redirectUrl} />
     }
 
+    const onRecordError = () => {
+        warningAlert.show({
+            msg: i18n.t('The dashboard could not be cached. Please try again.'),
```
Comment (cooper-joe) [inline src/pages/view/TitleBar/ActionsBar.js]: For consistency, I think the message `The dashboard couldn't be made available offline. Try again.` is clearer, as _caching_ is not referred to anywhere else.

**Review by cooper-joe — CHANGES_REQUESTED** [review]

File: src/pages/view/TitleBar/ActionsBar.js (hunk 1 above)
Comment (jenniferarnesen) [reply src/pages/view/TitleBar/ActionsBar.js]: @cooper-joe Should it also be a critical message rather than a warning?

Comment (jenniferarnesen) [reply src/pages/view/TitleBar/ActionsBar.js]: I updated the PR description with a screenshot showing the new text plus using critical

Comment (cooper-joe) [reply src/pages/view/TitleBar/ActionsBar.js]: Looks good 👍

**Review by cooper-joe — APPROVED** [review]

**Pull request merged**